sdkit/models/model_loader/stable_diffusion/__init__.py

```diff
@@ -253,7 +253,7 @@ def load_diffusers_model(
     # make the compel prompt parser object
     textual_inversion_manager = DiffusersTextualInversionManager(default_pipe)
     if is_sd_xl:
-        skip = Skip.PENULTIMATE_HIDDEN_STATES_NON_NORMALIZED if clip_skip else Skip.LAST_HIDDEN_STATES_NORMALIZED
+        skip = Skip.PENULTIMATE_HIDDEN_STATES_NON_NORMALIZED 
```

Comment:
Shouldn't this be `Skip.LAST_HIDDEN_STATES_NORMALIZED` if we're never clip-skipping for SDXL? Sorry if I'm missing something.

Reply:
According to Damian there is no supported clip skip currently https://github.com/damian0815/compel/issues/56. `Skip.PENULTIMATE_HIDDEN_STATES_NON_NORMALIZED` is a special state for SD XL. `LAST_HIDDEN_STATES_NORMALIZED` and `PENULTIMATE_HIDDEN_STATES_NORMALIZED` are only for SD 1.5/2.x (see https://github.com/damian0815/compel/blob/3f2f3efb772f4b997493e7757ecb91d8e6479657/src/compel/embeddings_provider.py#L49C1-L49C1 )